res/res_pjsip_pubsub.c

```diff
@@ -2491,6 +2529,13 @@ static int send_notify(struct sip_subscription_tree *sub_tree, unsigned int forc
 		pjsip_msg_add_hdr(tdata->msg, (pjsip_hdr *) require);
 	}
 
+	handler = subscription_get_handler_from_tdata(tdata, sub_tree->persistence->endpoint);
```

Comment:
The handler should already be accessible directly at:

sub_tree->root->handler

And guaranteed to be non-NULL